components/ActivityPage.tsx

```diff
@@ -1,10 +1,236 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, { useState, ChangeEvent } from 'react';
+import RNDateTimePicker from '@react-native-community/datetimepicker';
+import { daysOfTheWeek } from '../utils/constants';
+import {
+  StyleSheet,
+  View,
+  ViewStyle,
+  Text,
+  TextStyle,
+  TextInput,
+  TouchableOpacity
+} from 'react-native';
+
+interface Styles {
+  title: TextStyle;
+  container: ViewStyle;
+  label: TextStyle;
+  input: TextStyle;
+  inputWrapper: ViewStyle;
+  button: ViewStyle;
+  daysOfTheWeekContainer: ViewStyle;
+  selectedDaysOfWeek: ViewStyle;
+}
+
+interface IValues {
+  name?: string;
+  description?: string;
+  motivation?: string;
+  reminder?: string;
+  selectedDays?: any;
+  date?: any;
+}
+interface IProps {
+  navigation?: {
+    navigate: Function;
+  };
+}
+
+export default function ActivityPage(props: IProps) {
+  const [values, setValues] = useState<IValues>({ date: new Date() });
+  const [selectedDays, setSelectedDays] = useState<string[]>([]);
+
+  function handleChange(name: string, event: string): void {
+    setValues({
+      ...values,
+      [name]: event
+    });
+  }
+
+  function handleDayOfWeekClick(day: string) {
+    if (selectedDays.includes(day)) {
+      const unselect = selectedDays.filter(selectedDay => selectedDay != day);
```

Comment:
why is this named unselect?

And can we make the `!=` strict using `!==` or is there a reason why it's one `=`

Reply:
Also can you explain what the code here is trying to achieve 😖

Reply:
the if statement is supposed to remove or unselect the item that is already available and is being selected again. I agree to doing a strict